components/resources/library/src/androidMain/kotlin/org/jetbrains/compose/resources/AndroidContextProvider.kt

```diff
@@ -37,11 +38,34 @@ fun PreviewContextConfigurationEffect() {
     }
 }
 
+/**
+ * Sets the Android instrumented context.
+ *
+ * @param context The Android context to be set as the instrumented context.
+ *                This context will be used by a ResourceReader to read test assets.
+ *
+ * Example usage:
+ * ```
+ * fun configureTestEnvironment() {
+ *     setAndroidInstrumentedContext{
+ *         InstrumentationRegistry.getInstrumentation().context
+ *     }
+ * }
+ * ```
+ */
+@ExperimentalResourceApi
+fun setAndroidInstrumentedContext(context: Context) {
```

Comment:
```suggestion
fun setResourcesAndroidInstrumentedContext(context: Context) {
```

Better to not use a generic name for a resource function

Reply:
This API is android only (and it makes sense). 
I'm wondering how would developers write a common test in this case?

Do they need to create some expect/actual on their side?

Reply:
yes, the only way to read test resources on the android is provide the special context.
So, users are supposed to do it by their own

(or we can implement it in `compose-ui-tests` module, I guess)

Reply:
@igordmn the fuunction is in resource's package. I would say it is enough.
we already provide `PreviewContextConfigurationEffect` function with almost the same logic.